invenio_vocabularies/resources/resource.py

```diff
@@ -164,3 +135,37 @@ def launch(self):
         """Create a task."""
         self.service.launch(g.identity, resource_requestctx.data or {})
         return "", 202
+
+
+class VocabulariesAdminResource(RecordResource):
+    """Resource for vocabularies admin interface."""
+
+    def create_url_rules(self):
+        """Create the URL rules for the record resource."""
+        routes = self.config.routes
+        rules = super().create_url_rules()
+
+        rules.append(
+            route("GET", routes["all"], self.get_all_vocabulary_types),
+        )
```

Comment:
minor: if we replace `"all"` with `"list"`, then we can completely remove this method actually, and rename `get_all_vocabulary_types` to `search` (so that it maps correctly to the `GET /api/vocabularies` route rule.

Reply:
see above comment

Reply:
The parent's create_url_rules uses routes["items"] though, so we still need to overwrite this method in the VocabulariesAdminResource which uses "list".